### ISSUE TITLE
How to disable separator

### ISSUE THREAD

**Issue by donnygunawan93** [FCSAmerica/angular-fcsa-number#32]
Hello buddy ^_^,
I want to use this library for validate PIN,
so I don't need separator,
how to disable?

**Comment by Shawful** [comment]
agree. upvote.  I am attempting a similar angular validation on a credit card and the commas make me sad :(

**Comment by ninjamonz** [comment]
last commit 2014, I think this repo no longer mantained, check this one https://github.com/uhlryk/angular-dynamic-number

**Comment by Shawful** [comment]
I fixed it by adding a few minor lines of code and adding `fcsa-number="{noSeparator: true}"` to my markup

fcsa now has the ability to remove commas from result (for credit card numbers)
https://github.com/kixer-com/ads/commit/85200dba673374e8c98bd74932831c36f44a19a6
